src/server/endpoints/file/list.py

```diff
@@ -82,4 +86,15 @@ def action(project: Project, props: Optional[Props] = None) -> Result:
 
 IGNORED_FOLDERS = [
     "node_modules",
+    "src",
```

Comment:
@shashigharti 
I meant more really low-level specific stuff like `__pycache__` etc (check standard gitignore for python/javascript). We should not hide any normal user dirs like test etc